run.py

```diff
@@ -109,6 +110,11 @@ def run(args):
     if cleanup_name:
         pass  # TODO: cleanup cluster and skip test execution
 
+    # Check if we are running on windows and bail out sooner
+    if platform.system() == "Windows":
+        log.info("OS not supported by Openshift Installer")
+        return 1
```

Comment:
@vasukulkarni , I'm thinking about managed return codes like exceptions so we will be able to handle them outside of the test itself.
I still don't have a full vision of how to use it but think it will be handy in the future.
In this case,
ERRWINDOWS = <any non zero number>

return ERRWINDOWS

WDYT?

Reply:
@RazTamir yep that makes sense to me, we will have to build the list as we go, we can start with something small for now.

Reply:
And of course I forgot Petr comment:

```python
ERRWINDOWS = 15 # Any non zero return code

return ERRWINDOWS
```

Reply:
@RazTamir are you looking for something like an Enum here? I am experimenting with something similar for test status codes.

Ex.
```
class TestStatus(Enum):
    PASSED = 0
    FAILED = 1
    NOT_EXECUTED = 2
    SKIPPED = 3
```

Reply:
@clacroix12 I know brought this up today, I am thinking Enum as well, ex: TestExitErrorCodes, different from your class but can be in the same file.

Reply:
Yes @clacroix12